app/controllers/spree/admin/search_controller_decorator.rb

```diff
@@ -16,10 +16,17 @@ def known_users
     render :users
   end
 
+  def customers
```

Comment:
We will need to scope the list of customers searched to those of enterprises owned by the current user, to prevent a user from find a customer of another enterprise by submitting a different distributor_id in the params. We could just just do a test to check that `spree_current_user.enterprises.pluck(:id).include? params[:distributor_id]` and render an empty array if it doesn't?